app/src/main/java/org/eyeseetea/malariacare/data/database/iomodules/dhis/exporter/PushController.java

```diff
@@ -26,6 +26,7 @@
 import org.eyeseetea.malariacare.data.database.iomodules.dhis.importer.models.EventExtended;
 import org.eyeseetea.malariacare.data.database.model.Survey;
 import org.eyeseetea.malariacare.data.database.utils.PopulateDB;
+import org.eyeseetea.malariacare.data.remote.PullDhisSDKDataSource;
 import org.eyeseetea.malariacare.receivers.AlarmPushReceiver;
 import org.eyeseetea.malariacare.data.remote.SdkController;
```

Comment:
@idelcano PushController invoked register and unregister methods of SDKController and this methods only made sense with the old sdk